src/components/Footer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Footer.scss';
+
+const Footer = () => {
+    return (
+        <div className="background">
+            <div className="text">
+                <p>
+                    <a
+                        class="text link-offset-2 link-underline link-underline-opacity-0"
```

Comment:
Attribute class is not allowed here
`.jsx`에서는 class를 사용하는 것을 허용하지 않습니다.
```suggestion
                        className={
                            'text link-offset-2 link-underline link-underline-opacity-0'
                        }
```

* [참고자료](https://www.codecademy.com/courses/react-101/lessons/react-jsx-advanced/exercises/jsx-classname-class)